packages/frontend/src/components/SchedulersView/SchedulersView.tsx

```diff
@@ -46,7 +46,8 @@ const Schedulers: FC<SchedulersProps> = ({
     dashboards,
 }) => {
     const { classes, theme } = useTableStyles();
-    const { data: allSlackChannels } = useSlackChannels();
+    const { data } = useGetSlack();
+    const { data: allSlackChannels } = data?.slackEnabled ? useSlackChannels() : { data: [] };
```

Comment:
React Hooks must be called in the exact same order in every component render. They cannot be called inside a condition. 

Instead, the hook can have an argument and pass it to `useQuery` hook.

```
return useQuery({
        queryKey,
        queryFn: () => ...,
        enabled: !!isSlackInstalled,
    });
```